ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Docker/tasks/main.yml

```diff
@@ -3,7 +3,7 @@
 # Docker #
 ##########
 - name: Test if docker is already installed
-  shell: docker >/dev/null 2>&1
+  shell: docker version 2>&1 | grep Version | awk '{print $2}'
```

Comment:
Not sure why the change from "is it installed?" to "obtain a version if installed".  There doesn't appear to be any use of the version number that I see.  Is that check failing as is?

Reply:
The current check is just failing as is. The command uses docker version command and if there is an output that means docker is installed. 
![image](https://user-images.githubusercontent.com/39133984/53421806-4a95cc00-39ac-11e9-9a08-ed46336f0674.png)

Reply:
The previous method was checking the return code of the command.  `echo $?` after `docker >/dev/null 2>&1` should give a `0` no?  Or maybe this is something that works though a shell and not ansible??

Reply:
I just realized I messed up when testing. I thought that the docker check wasn't working, but it was because of other parameters that weren't there. I will revert it back